semantics/executable-spec/src/Data/Closure.hs

```diff
@@ -0,0 +1,140 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE StandaloneDeriving  #-}
+{-# LANGUAGE DerivingVia #-}
+
+-- | Supports the creation of objects with type (Closure name env (a -> b)) which are serializable functions.
+module Data.Closure
```

Comment:
Can we remove this module now it's not being used?

Reply:
Done